webapp/src/client/client.js

```diff
@@ -15,30 +15,16 @@ export default class Client {
         return this.doGet(`${this.url}/connected?reminder=${reminder}`);
     }
 
-    getReviews = async () => {
-        return this.doGet(`${this.url}/reviews`);
-    }
-
-    getYourPrs = async () => {
-        return this.doGet(`${this.url}/yourprs`);
-    }
+    getSidebarContent = async () => this.doGet(`${this.url}/sidebar-content`);
```

Comment:
To be consistent with the rest of the file

```suggestion
    getSidebarContent = async () => {
        return this.doGet(`${this.url}/sidebar-content`);
    }
```